packages/dx-react-grid-bootstrap3/src/templates/table-header-cell.jsx

```diff
@@ -54,12 +54,12 @@ export class TableHeaderCell extends React.PureComponent {
       <th
         style={{
           position: 'relative',
-          ...(showSortingControls || draggingEnabled ? {
+          ...(sortingEnabled || draggingEnabled ? {
             userSelect: 'none',
             MozUserSelect: 'none',
             WebkitUserSelect: 'none',
           } : {}),
-          ...(showSortingControls || draggingEnabled ? { cursor: 'pointer' } : null),
+          ...(sortingEnabled || draggingEnabled ? { cursor: 'pointer' } : null),
```

Comment:
And here?

Reply:
It's because sorting control can be rendered as an inactive element. If sorting control is active we add specific style `cursor: pointer`. But, probably, it's better to use `(showSortingControls && sortingEnabled) || draggingEnabled`...